nightwatch.conf.js

```diff
@@ -31,7 +31,7 @@ const LDAP_BASE_DN = process.env.LDAP_BASE_DN || 'cn=admin,dc=owncloud,dc=com'
 const LDAP_ADMIN_PASSWORD = process.env.LDAP_ADMIN_PASSWORD || 'admin'
 const OCIS_SKELETON_DIR = process.env.OCIS_SKELETON_DIR || './tests/testing-app/data/webUISkeleton/'
 const OPENID_LOGIN = RUN_ON_OCIS || !!process.env.OPENID_LOGIN
-const WEB_UI_CONFIG = process.env.WEB_UI_CONFIG || path.join(__dirname, 'dist/config.json')
+const WEB_UI_CONFIG = process.env.WEB_UI_CONFIG || path.join(__dirname, 'config/config.json')
```

Comment:
why do we need this change? in https://github.com/owncloud/web/pull/4740/files#diff-6a968d4b7901c8526c8d9a86d1f434737dbfe2b00da21a7d3041ed4e3d58d62eR1055 you copy the config file to `dist`

Reply:
you're right, that's not required, I'll change that